composer.json

```diff
@@ -59,6 +59,7 @@
 	},
 	"extra": {
 		"typo3/cms": {
+			"extension-key": "bootstrap_package",
```

Comment:
@sypets you are right, the key is missing, but it should be calendarize_news

Reply:
sorry about that. Thanks for merging.